frontend/catalyst/jax_primitives.py

```diff
@@ -712,6 +689,7 @@ def lower_qnode_to_funcop(ctx, callable_, call_jaxpr):
     name = "module_" + callable_.__name__
     # pylint: disable-next=no-member
     with NestedModule(ctx, name) as module, ir.InsertionPoint(module.regions[0].blocks[0]) as ip:
+        _transform_named_sequence_lowering(ctx)
```

Comment:
won't this create multiple named sequences, one for each apply_pass?

Reply:
Never mind, I was fooled by github's collapsed diff (it made it look like this line belonged to apply_pass primitive's lowering) :sweat_smile: